plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreProcedure.java

```diff
@@ -118,6 +120,8 @@ enum TransitionModifies {
     }
 
     private long oid;
+    private long xmin;
+    private boolean procWasCreatedOrReplaced;
```

Comment:
What's the point of the `proc` prefix if we're already inside the class called `PostgreProcedure`? Please remove it